docs/flutter/v2/features/background-handling.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Background Handling
+nav: 3.28
+---
+
+While a user has joined a 100ms Room they can put the app in Background Mode & usually expect a certain subset of app functionality continues to work. 
+
+Background Mode implies that the application is not currently visible to the user, not responding to user input, and running in the Background. 
+
+Background state in Flutter apps is represented by `AppLifecycleState.paused` as described [here](https://api.flutter.dev/flutter/dart-ui/AppLifecycleState.html#paused).
+
+Android & iOS have different mechanisms of handling app behaviour which includes limiting access to Camera & Microphone, time allowed to keep running before getting  completely stopped, Aggressive Battery Optimisations on Android OEMs, etc.
+
+There are 2 prominent behaviours that are mainly affected when apps transition to Background State - Publishing Audio/Video, and Playing incoming Audio/Video.
+
+Let's breakdown how your apps can implement efficient handling of Background states on Android & iOS Platforms.
+
+## Android
```

Comment:
- Many scenarios can happen here
- Use  1 or more Table to show behaviours
- Add more prominent differences between playing or capturing, audio or video, app is in foreground or background, Android/iOS, Android 11+ / below 11, etc